package.json

```diff
@@ -42,7 +42,7 @@
     "npm": "^5.0.4",
     "phantomjs": "^2.1.7",
     "phantomjs-prebuilt": "^2.1.16",
-    "puppeteer": "^0.13.0",
+    "puppeteer": "^1.3.0",
```

Comment:
Isn't the a lockfile change for this? It doesn't effect the module but you could commit that as part of this change otherwise it will show up as changed files the next time someone installs modules.

Reply:
Oh I used npm not yarn that's why it didn't update it. Will do,

Reply:
so with npm install it works - but with yarn  the installation is failing :

```
error /home/sg/git/gulp-jasmine-browser/node_modules/slimerjs: Command failed.
Exit code: 1
Command: node install.js
Arguments: 
Directory: /home/sg/git/gulp-jasmine-browser/node_modules/slimerjs
Output:
Considering SlimerJS found at /usr/bin/slimerjs
Found SlimerJS at /usr/bin/slimerjs ...verifying
Error verifying slimerjs, continuing { Error: Command failed: /usr/bin/slimerjs --version

    at ChildProcess.exithandler (child_process.js:289:12)
    at ChildProcess.emit (events.js:182:13)
    at maybeClose (internal/child_process.js:962:16)
    at Socket.stream.socket.on (internal/child_process.js:381:11)
    at Socket.emit (events.js:182:13)
    at Pipe._handle.close (net.js:606:12)
  killed: false,
  code: 1,
  signal: null,
  cmd: '/usr/bin/slimerjs --version' }
Slimer installation failed TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received type undefined
    at assertPath (path.js:39:11)
    at Object.join (path.js:1155:7)
    at findSuitableTempDirectory (/home/sg/git/gulp-jasmine-browser/node_modules/slimerjs/install.js:113:30)
    at /home/sg/git/gulp-jasmine-browser/node_modules/slimerjs/install.js:408:19
    at nextTickCallback (/home/sg/git/gulp-jasmine-browser/node_modules/kew/kew.js:47:28)
    at process._tickCallback (internal/process/next_tick.js:61:11) TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received type undefined
    at assertPath (path.js:39:11)
    at Object.join (path.js:1155:7)
    at findSuitableTempDirectory (/home/sg/git/gulp-jasmine-browser/node_modules/slimerjs/install.js:113:30)
    at /home/sg/git/gulp-jasmine-browser/node_modules/slimerjs/install.js:408:19
```

(and the same error with slimerjs. I installed phantom globally but slimerjs won't run on my system because my firefox version is too new. the yarn command error this time was: 

```
error /home/sg/git/gulp-jasmine-browser/node_modules/slimerjs: Command failed.
Exit code: 1
Command: node install.js
Arguments: 
Directory: /home/sg/git/gulp-jasmine-browser/node_modules/slimerjs
Output:
Considering SlimerJS found at /usr/bin/slimerjs
Found SlimerJS at /usr/bin/slimerjs ...verifying
Error verifying slimerjs, continuing { Error: Command failed: /usr/bin/slimerjs --version

    at ChildProcess.exithandler (child_process.js:289:12)
    at ChildProcess.emit (events.js:182:13)
    at maybeClose (internal/child_process.js:962:16)
    at Socket.stream.socket.on (internal/child_process.js:381:11)
    at Socket.emit (events.js:182:13)
    at Pipe._handle.close (net.js:606:12)
  killed: false,
  code: 1,
  signal: null,
  cmd: '/usr/bin/slimerjs --version' }
Slimer installation failed TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received type undefined
    at assertPath (path.js:39:11)
    at Object.join (path.js:1155:7)
    at findSuitableTempDirectory (/home/sg/git/gulp-jasmine-browser/node_modules/slimerjs/install.js:113:30)
    at /home/sg/git/gulp-jasmine-browser/node_modules/slimerjs/install.js:408:19
    at nextTickCallback (/home/sg/git/gulp-jasmine-browser/node_modules/kew/kew.js:47:28)
    at process._tickCallback (internal/process/next_tick.js:61:11) TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received type undefined
    at assertPath (path.js:39:11)
    at Object.join (path.js:1155:7)
    at findSuitableTempDirectory (/home/sg/git/gulp-jasmine-browser/node_modules/slimerjs/install.js:113:30)
    at /home/sg/git/gulp-jasmine-browser/node_modules/slimerjs/install.js:408:19
```

I don't know why yarn is assuming I have both phantomjs and slimerjs installed globally (or in my path) probably I should investigate more but unfortunately I don't have the time. Could you please update that file for me ? just in case this is my system

arch linux
node -v 
10.13.0
npm -v 
6.4.1
yarn -v 
1.10.1

I think this issue is important. In my case I need a newer puppeteer version than 0.13.0 because that browser doesn't support some features I need in my project. Since console.log doesn't work and the stack thrown when there's a jasmine expect failure don't say anything (perhaps another or same issue? ) headless is not useful for debugging (and you must use the server) - only for run green tests in CI. 

Also I think puppeteer latest won't work, that's why I updated to latest - 1. In that case we should also document in readme explicitly which pupetter/phantomjs/slimerjs versions users must install. I can do it in a separate pull request after this is merged.

Reply:
Sure thing I’ll give it a go.

Reply:
i generated the lockfile, for some reason i could not attach it as a comment so I'll just send you a PR on your original branch